entity-management-definitions/pom.xml

```diff
@@ -55,6 +55,11 @@
 			</exclusions-->
         	
         </dependency>
+       	<dependency>
```

Comment:
I have added the validation classes in the entity-management-definitions module, the package eu.europeana.entitymanagement.normalization. Since there is a cyclic dependency between the validation classes and the entity classes, the validation classes should be in the same module as the definition classes. The logger is preserved to log some exceptions from the validation module because the class cannot throw Exceptions since it is predefined.

Reply:
No ... the definitions model contains only the domain objects but no business logic, the business logic is typically implemented  in the web and utils modules. Please update the code as requested...